packages/wallets/src/AppContent.tsx

```diff
@@ -1,8 +1,13 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { defineViewportHeight } from './utils/utils';
 import { Router } from './routes';
 import './AppContent.scss';
 
 const AppContent: React.FC = () => {
+    useEffect(() => {
+        defineViewportHeight();
```

Comment:
NOTE: Trigger function here because we need innerHeight for both wallets and cashier routes